src/openforms/conf/base.py

```diff
@@ -1027,13 +1027,11 @@
     "'self'",
 ] + config("CSP_EXTRA_DEFAULT_SRC", default=[], split=True)
 
-# CORS_ALLOWED_ORIGINS is included because we (likely) need to redirect back to those
-# third party domains that are embedding the SDK after login. Chrome in particular
-# validates the entire redirect chain, see: https://stackoverflow.com/a/69439102
-CSP_FORM_ACTION = (
-    ["'self'"]
-    + CORS_ALLOWED_ORIGINS
```

Comment:
I don't think that dropping this is a good idea, especially if for example `http://localhost:3000` is listed in `CORS_ALLOWED_ORIGIN` (which I have in my local `.env`).

Dropping the support for `CSP_EXTRA_FORM_ACTION` is also a breaking change, someone may be using this. I think this should also stay for people that want to *add* to the list instead of replacing it alltogether. Since it doesn't appear to be documented, we can treat this as non-public API, but then there should be an entry in the changelog for devops engineers on how to update their infra accordingly.

Reply:
Yes I dropped it because I saw it wasn't documented (unlike the others). The only issue I see is if people want to replace the whole list (if for instance they don't want `https:`). Maybe we should do something like:

```python
CSP_FORM_ACTION = config(
    "CSP_FORM_ACTION",
    default=["\"'self'\"", "https:"] + config("CSP_EXTRA_FORM_ACTION", default=[], split=True),
    split=True,
) + CORS_ALLOWED_ORIGINS
```

But is overly complex.

Reply:
and flawed too because it still adds `CORS_ALLOWED_ORIGINS` anyway.

Reply:
So what's the path for this one then? I don't feel like dropping `CSP_EXTRA_FORM_ACTION` is a good idea, it creates an inconsistent state with the other csp settings where such an extra env variable exists.

Reply:
probably that suggestion above is the best?

* for dev, I can add `CSP_EXTRA_FORM_ACTION=http:`
* in prod if I want to add additional domains, infra can use `CSP_EXTRA_FORM_ACTION`
* in prod, if they want to make it more strict, they set `CSP_FORM_ACTION` envvar

Reply:
Hum, then if we need to support both `CSP_EXTRA_FORM_ACTION` and `CSP_FORM_ACTION` we need to do something similar to https://github.com/open-formulieren/open-forms/pull/4223#discussion_r1589167342.

But supporting extension of settings is hard (what happens if both `CSP_FORM_ACTION` and `CSP_EXTRA_FORM_ACTION` are set?)